extension/encoding/jsonlogencodingextension/README.md

```diff
@@ -11,4 +11,33 @@
 [development]: https://github.com/open-telemetry/opentelemetry-collector#development
 <!-- end autogenerated section -->
 
-The `jsonlog` encoding extension is used to marshal/unmarshal JSON log body, ignoring other log fields.
\ No newline at end of file
+The `jsonlog` encoding extension is used to marshal/unmarshal JSON log body, ignoring other log fields.
+
+## Configuration
+
+| Name                     | Description                                     | Default                                      |
+| ------------------------ | ----------------------------------------------- | -------------------------------------------- |
+| raw_log                  | Ignore all attributes and just get body of log  | true                                         |
+
+
+
+### raw_log
+
+The `raw_log` property when set to false will grab resource and attributes and add them as key value pairs to the json body iterating through all the logs and create a json array looking along the lines of:
+
+```json
+[
+  {
+    "body": "{\"log\": \"test\"}",
```

Comment:
AFAICT your logic will append the body as is, not encode it to string

Reply:
So this test here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32722/files#diff-ff7736db9322854a955a13662c43bf9b8e2212a8aed9ef7188c0bf7507a49453R58 I think validates that the behavior will be to encode it to a string.

Reply:
String to String will show this, but try setting the body to be a map.

Reply:
That was indeed my error thanks for catching this!